client/src/modules/roles/roles.html

```diff
@@ -7,7 +7,7 @@
 
     <div class="toolbar" ng-show="RolesCtrl.canEditRoles">
       <div class="toolbar-item">
-        <button class="btn btn-default" ng-click="RolesCtrl.add()" data-method="create">
+        <button class="btn btn-default" ng-click="RolesCtrl.createUpdateRoleModal()" data-method="create">
```

Comment:
We should probably standardize this nomenclature.  My former Python self agrees "explicit is better than implicit".